hrjobcontract/js/src/job-contract/services/utils.service.js

```diff
@@ -72,6 +73,40 @@ define([
         return deffered.promise;
       },
 
+      getOptionValues: function (groupNames) {
+        var deferred = $q.defer();
+        var multiple = _.isArray(groupNames);
+
+        CRM.api3('OptionValue', 'get', {
```

Comment:
from looking at the other methods in this service, they are all using `API.resource`, so should we use that instead of relying on `CRM.api3`. Let's try to have consistent code